bootstrap/web/iframe-adapter.js

```diff
@@ -50,12 +50,6 @@ let messageHandler = function(message) {
                     delete ZoweZLUX.iframe.contextMenuActions[key];
                 }
                 return;
-            case 'viewportEvents.spawnContextMenu':
-                if(data.contextMenuItemIndex !== undefined){
-                    ZoweZLUX.iframe.contextMenuActions[key][data.contextMenuItemIndex].action();
-                    delete ZoweZLUX.iframe.contextMenuActions[key];
-                }
-                return;
```

Comment:
Was this intentionally removed? I think this was added to make the Zowe context menu work for Iframes

Reply:
Its a duplicate

Reply:
https://github.com/zowe/zlux-app-manager/pull/302/files#diff-083b8438c11bd78ce9a7db4a2dd96b4406bed2405ae89f8ae054bc02b4d05dc9R47